### ISSUE TITLE
"Does traffic_calming still exist" quest needs to be more clear

### ISSUE THREAD

**Issue by lightonflux** [streetcomplete/StreetComplete#2969]
**How to Reproduce**

1. find the quest
2. read the quest
3. have a lack of understanding what narrow / wide (de: schmal & breit) means in the context of traffic calming
4. Confirm the wrong type of traffic calming

**Versions affected**
v32.1

**Remedy**
Show pictures of the type of traffic calming, so that the user can more easily confirm/deny if e.g. a bump is still present and not replaced with a table.

**Context**
In the last decade an area here was rebuilt to feature traffic tables, this meant that several bumps were replaced. 
Without having tagged traffic calming before i had to check with the node data first, what the meaning of "schmal" (narrow) and "breit" (wide) (quest tranlation in DE) means in this context. And i only checked because two tables were suggested, both as narrow and wide, which made me realize the disparity.

**Comment by lightonflux** [comment]
Additionally, the current german translation might be wrong. Because it references the width, but in the wiki the depth differenciates a bump from a table. https://wiki.openstreetmap.org/wiki/DE:Key:traffic_calming

**Comment by matkoniecz** [comment]
See https://github.com/openstreetmap/iD/blob/develop/CONTRIBUTING.md#translating for how translation can be fixed (StreetComplete is using here iD presets)

**Comment by DerDings** [comment]
> Additionally, the current german translation might be wrong. Because it references the width, but in the wiki the depth differenciates a bump from a table. https://wiki.openstreetmap.org/wiki/DE:Key:traffic_calming

Width and depth depends on the perspective. SC usually does assume pedestrians perspective, which usually (but not always) is looking from the sidewalk at the road.

**Comment by westnordost** [comment]
-> So, needs to be solved in the translation of iD presets.

**Comment by peternewman** [comment]
So I've just solved a few of these, and it does feel these are a bit questionable even in English, which seems to effectively use the tag names. I'm not sure there could be any improvement of them as preset names in iD, but for this quest, having a description/picture or some further detail would probably be helpful.

I can clearly tell (in the unlikely event) the traffic calming has been removed, but I'm at risk of saying something is still a hump when it's actually been replaced with a table, or it was perhaps miss-tagged in the first place. In comparison, despite not owning a dog, I know what a dog waste bin looks like, and indeed have added a note in the past when something was tagged as a rubbish bin or vice versa.

The `traffic_calming`, and probably `advertising ~ column|board|poster_box` seem like the only ones in the existence quest where I might need to refer to the wiki to realise it had changed. I could also imagine columns and boards being upgraded to screens to earn the company more money and with less effort required.

Maybe this would be another candidate for @smichel17 's #1913 .
